add flag for regular schools, revert radio buttons

touchRED opened the pull request:
Closes #502, reverting school type radio buttons and adding a flag for "regular" schools (non-charter, non-magnet, non-bie).

Viewable [here](https://inspiring-lumiere-20b37d.netlify.app/).

Notes:

- Hard to tell if my "regular" `rg` flag is working properly – there are some schools like "Sam H. Lawson Middle" that have an `rg` flag but still show up when the "regular" checkbox is unchecked. But unchecking this box should render only charter/magnet schools, right? Maybe I'm misunderstanding the logic of the checkboxes.

Lane reviewed (changes requested):
@touchRED this looks right to me, i just checked and "Sam H. Lawson Middle" does not show up when the "Regular" box is unchecked. if you have previously selected (or "pinned") the location, it will show though.  that is expected.

the only change needed here is that "BIE" should also be a school type, instead of an exclusive filter in its own group

![image](https://user-images.githubusercontent.com/21034/106644176-9d387500-653f-11eb-848d-179a938516a8.png)

you can see this locally if you add

```
REACT_APP_EMBARGOED=1
REACT_APP_SHOW_PR=1
```

to your `.env.local`

touchRED commented:
@Lane this makes sense, I modified the flags structure to include `bie` in the same group as charter, magnet, etc. I also removed `bie` from `EXCLUSIVE_FLAGS` – lmk if that isn't the plan!
